lib/segment/src/vector_storage/query_scorer/mod.rs

```diff
@@ -11,3 +16,14 @@ pub trait QueryScorer<TVector: ?Sized> {
 
     fn score_internal(&self, point_a: PointOffsetType, point_b: PointOffsetType) -> ScoreType;
 }
+
+pub fn score_multi<TMetric: Metric>(a: &MultiDenseVector, b: &MultiDenseVector) -> ScoreType {
```

Comment:
maybe you could have a comment on the function about the `MaxSim` operator and link towards a definition.

Reply:
added a comment with link to metric definition